tests/test_cognitive_complexity.py

```diff
@@ -142,9 +142,9 @@ def f(a):
         def foo(a):
             if a:  # +2
                 return 1
-        bar = lambda a: lambda b: b or 2  # +2 (+2 for or because lambda increases nesting)
+        bar = lambda a: lambda b: b or 2  # +1
```

Comment:
Not sure if this is correct and related to the fix.

Reply:
I'm interpreting this as a sequence of logical operators in a return statement (per page 7) -- and the core of the PR is that these sequences do not get a nesting increment.

The only alternative I see -- interpreting this as equivalent to a ternary (b if b else 2) -- would mean counting this statement as +3 (+2 for the nesting levels from the lambdas, +1 for the ternary itself). That could be a difficult special case to implement.

What do you think?